runtime/src/lib.rs

```diff
@@ -240,9 +240,30 @@ impl pallet_sudo::Config for Runtime {
 
 impl pallet_ethereum_chain_id::Config for Runtime {}
 
+/// Current approximation of the gas/s consumption considering
+/// EVM execution over compiled WASM (on 4.4Ghz CPU).
+/// Given the 500ms Weight, from which 75% only are used for transactions,
+/// the total EVM execution gas limit is: GAS_PER_SECOND * 0.500 * 0.75 => 3_000_000.
+pub const GAS_PER_SECOND: u64 = 8_000_000;
+
+/// Approximate ratio of the amount of Weight per Gas.
+/// u64 works for approximations because Weight is a very small unit compared to gas.
+pub const WEIGHT_PER_GAS: u64 = WEIGHT_PER_SECOND / GAS_PER_SECOND;
+
+pub struct MoonbeamGasWeightMapping;
```

Comment:
I like the integration tests. I wouldn't mind some simple Rust unit tests too at least at the edge cases. Like what do we expect 0 gas to map to, or 0 weight.

I guess it might make sense to write those once we've refined this a little bit though. I'm fine with this PR going in without these unit tests yet.

Reply:
Agree. I'll probably go for it once the computation gets a bit more complex